src/index.html

```diff
@@ -44,5 +44,154 @@ <h2 class="headline-2">I’m a software developer</h2>
+                            <div class="language"><p>html</p></div>
+                        </div>
+                        <button>See Project</button>
+                    </div>
+
+                </div>
+                <div class="card">
+                    <div class="card_info">
+                        <h5>Multi-Post Stories Gain+Glory</h5>
+                        <div class="card_languages">
+                            <div class="language"><p>Ruby on rails</p></div>
+                            <div class="language"><p>css</p></div>
+                            <div class="language"><p>JavaScript</p></div>
+                            <div class="language"><p>html</p></div>
+                        </div>
+                        <button>See Project</button>
+                    </div>
+
+                </div>
+            </div>
+        </section>
+        <section class="About_myself">
+            <div class="about_me">
+                <h3>About me</h3>
+                <div class="supporting_text">
+                     <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                <button>Get My Resume</button>
+            </div>
+
+            <ul class="about_me_cards">
+               <li> <div class="aboutme_card">
+                    <div class="languages_aboutme_img">
+                        <img src="../media/images/aboutme_languages.png" alt="languages icon"/>
+                    </div>
+                    <div class="languages_section">
+                        <p>Languages</p>
+                        <div class="languages_list">
+                            <div class="language"><p>JavaScript</p></div>
+                            <div class="language"><p>Ruby</p></div>
+                            <div class="language"><p>html</p></div>
+                            <div class="language"><p>css</p></div>
+                        </div>
+                    </div>
+                    
+                </div></li>
+                
+                <li> <div class="aboutme_card">
+                    <div class="languages_aboutme_img">
+                        <img src="../media/images/frameworks.png" alt="languages icon"/>
+                    </div>
+                    <div class="languages_section">
+                        <p>Frameworks</p>
+                        <div class="languages_list">
+                            <div class="language"><p>Bootstrap</p></div>
+                            <div class="language"><p>Ruby on rails</p></div>
+                            <div class="language"><p>RSpec</p></div>
+                            <div class="language"><p>Capybara</p></div>
+                            <div class="language"><p>Selenium</p></div>
+                        </div>
```

Comment:
- [ ] [REQUIRED] Kindly refer to my comment concerning the language list been embedded in a `ul` and `li` tags.

Reply:
Addressed 🔥😎